File: Source/UnitTests/TestCode/MemberMapperTest/BaseBaseTypeMethod.java
package Test;
import junit.framework.TestCase;
public class A extends TestCase
{}
public class B extends A
{
	public void testAssertAreEquals()
	{
		double expected;
		double actual;
		assertEquals(expected, actual, 0);
		assertEquals(expected, actual, 0.01);
		StringBuffer sb;
		StringBuffer sf = sb.append(".");
		assertEquals(sb, sf);
	}
}

File: Source/UnitTests/TestCode/MemberMapperTest/BaseBaseTypeMethod.cs
namespace Test
{
	using junit.framework.TestCase;
	public class A : TestCase
	{}
	public class B : A
	{
		public void testAssertAreEquals()
		{
			double expected;
			double actual;
			NUnit.Framework.Assert.AreEqual(expected, actual, 0);
			NUnit.Framework.Assert.AreEqual(expected, actual, 0.01);
			StringBuffer sb;
			StringBuffer sf = sb.Append(".");
			NUnit.Framework.Assert.AreEqual(sb, sf);
		}
	}
}